pymeasure/adapters/serial.py

```diff
@@ -72,37 +72,39 @@ def _write_bytes(self, content, **kwargs):
         """Write the bytes `content` to the instrument.
 
         :param bytes content: The bytes to write to the instrument.
-        :param kwargs: Keyword arguments for the connection itself.
+        :param \\**kwargs: Keyword arguments for the connection itself.
         """
         self.connection.write(content, **kwargs)
 
     def _read(self, **kwargs):
         """Read up to (excluding) `read_termination` or the whole read buffer.
 
-        :param kwargs: Keyword arguments for the connection itself.
+        :param \\**kwargs: Keyword arguments for the connection itself.
         :returns str: ASCII response of the instrument (read_termination is removed first).
         """
-        read = self._read_bytes(-1, **kwargs).decode()
+        read = self._read_bytes(-1, break_on_termchar=True, **kwargs).decode()
         # Python>3.8 this shorter form is possible:
```

Comment:
One thing that's not 100% clear (for all methods, not specifically here): If it stops reading at the termination char, is the termination char included in the reply, or stripped? If the latter, I think we don't need the if-branch here anymore?

Reply:
The docstring of `read_bytes` says ":returns:.... (including termination)", makes it clearer?

Reply:
Oops, missed that.